exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillResultSetImpl.java

```diff
@@ -1334,8 +1335,6 @@ public String getQueryId() throws SQLException {
 
   @Override
   protected DrillResultSetImpl execute() throws SQLException{
-    DrillConnectionImpl connection = (DrillConnectionImpl) statement.getConnection();
-
     connection.getClient().runQuery(QueryType.SQL, this.prepareResult.getSql(),
```

Comment:
I don't know why the code was like that.  (Maybe to be symmetric with connection.getDriver(), or maybe from before client was kept?)

Eliminated redundant call to getClient().